h2o-docs/src/product/data-science/anova_glm.rst

```diff
@@ -11,134 +11,172 @@ Since ANOVA GLM is mainly used to investigate the contribution of each predictor
+Algorithm-specific parameters
+'''''''''''''''''''''''''''''
 
--  `ignore_const_cols <algo-params/ignore_const_cols.html>`__: Specify whether to ignore constant
-   training columns, since no information can be gained from them. This option is defaults to true (enabled).
+- **save_transformed_framekeys**: Set to ``True`` to save the keys of transformed predictors and interaction column. Defaults to ``False``.
 
--  `seed <algo-params/seed.html>`__: Specify the random number generator (RNG) seed for
-   algorithm components dependent on randomization. The seed is
-   consistent for each H2O instance so that you can create models with
-   the same starting conditions in alternative configurations. This value defaults to -1 (time-based random number).
+- **highest_interaction_term**: This limits the number of interaction terms (i.e. ``2`` means interaction between 2 columns only, ``3`` for three columns, etc.). Defaults to ``2``.
 
--  `score_each_iteration <algo-params/score_each_iteration.html>`__: (Optional) Specify whether to score
-   during each iteration of the model training. This value is disabled by default.
+- **nparallelism**: Number of models to build in parallel. Adjust according to your system. Defaults to ``4``. 
 
--  `offset_column <algo-params/offset_column.html>`__: Specify a column to use as the offset.
+- **type**: Refer to the SS type 1, 2, 3, or 4. We are currently only supporting type 3.
 
--  `weights_column <algo-params/weights_column.html>`__: Specify a column to use for the observation
-   weights, which are used for bias correction. The specified
-   ``weights_column`` must be included in the specified
-   ``training_frame``. 
-   
-    *Python only*: To use a weights column when passing an H2OFrame to ``x`` instead of a list of column names, the specified ``training_frame`` must contain the specified ``weights_column``. 
-   
-    **Note**: Weights are per-row observation weights and do not increase the size of the data frame. This is typically the number of times a row is repeated, but non-integer values are supported as well. During training, rows with higher weights matter more, due to the larger loss function pre-factor.
+Maybe algorithm-specific
+''''''''''''''''''''''''
```

Comment:
The following parameters are specific to GLM, GAM, Modelselection, ANOVAGLM.  So not sure how you want to call them.